modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -223,26 +219,33 @@ class ConnectionAvgStats {
             return;
         }
 
+        if (!data.transport || !data.transport.length) {
```

Comment:
having this return here makes me feel that if we add something else below that does not rely on transport we may have a bug